hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NodeExecutor.java

```diff
@@ -47,6 +47,7 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
     executorInfo = taskInfo.getExecutor();
     task = new Task(taskInfo);
     startProcess(driver, task);
+    new Thread(new TaskSupervisor(driver, task)).start();
     driver.sendStatusUpdate(TaskStatus.newBuilder()
         .setTaskId(taskInfo.getTaskId())
```

Comment:
If we're going to change this in NodeExecutor, we should also do this in NameNodeExecutor.